### DISCUSSION TITLE
Support for Odroid H2

### DISCUSSION THREAD

**Discussion by maureis** [adafruit/Adafruit_Blinka#1062] [New Board Requests]
I'm trying to use the i2c (with home assistant) but get the "NotImplementedError: Board not supported"). Can you add support for the Odroid H2?
Thank you.

**Comment by makermelissa** [comment]
I don't currently have the hardware, but it would be a nice board to add.

**Comment by makermelissa** [comment]
If you'd like to add it yourself, you can use these guides:
https://learn.adafruit.com/adding-a-single-board-computer-to-platformdetect-for-blinka
and
https://learn.adafruit.com/adding-a-single-board-computer-to-blinka
